### PR TITLE
build: support seplia testnet

### PR DESCRIPTION
**Pull request by TTNguyenDev**
image

### CODE REVIEW

File: packages/contracts/script/Verify.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import { exec } from "child_process";
+
+const verifyArbitrationToken = async (address: string, path: string) => {
+  exec(
+    ["forge verify-contract", address, path, "--constructor-args", "000000000000000000000000000000000000000000000000000000000000006000000000000000000000000000000000000000000000000000000000000000a00000000000000000000000000000000000000000000000000000000000000012000000000000000000000000000000000000000000000000000000000000000b436f75727420546f6b656e00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000024354000000000000000000000000000000000000000000000000000000000000", "--chain-id", 11155111, "--etherscan-api-key", "6EI6Z7T8UAAVRIY8JXRXKVKM5A1WH8PURD"].join(" "),
```
Comment (aahna-ashina) [inline packages/contracts/script/Verify.ts:5]: Where are these `--constructor-args` coming from?

**Review by aahna-ashina — COMMENTED** [review]
Can you also add some notes to the `README` how to use the commands in `Verify.ts`?

File: packages/contracts/script/Verify.ts (hunk 1 above)
Comment (TTNguyenDev) [reply packages/contracts/script/Verify.ts:5]: We can get it on etherscan. But yeah, I sent a wrong token argument, so I need to deploy again and update README file as well

**Review by aahna-ashina — APPROVED** [review]

File: packages/contracts/script/Verify.ts [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    (error, stdout, stderr) => {
+      console.log(error)
+      console.log(stdout)
+      console.log(stderr)
+    }
+  );
+}
+const main = async () => {
+  console.log("Contract verification...")
+  await verifyArbitrationToken("0x23Ca3002706b71a440860E3cf8ff64679A00C9d7", "lib/solmate/src/test/utils/mocks/MockERC20.sol:MockERC20")
```
[the remark this answers is not in the document]
Comment (TTNguyenDev) [reply packages/contracts/script/Verify.ts:37]: Yeah, sure

**Comment by TTNguyenDev** [comment]
@aahna-ashina please verify this PR again

**Review by aahna-ashina — APPROVED** [review]

**Pull request merged**